dataprofiler/profilers/float_column_profile.py

```diff
@@ -28,7 +30,22 @@ def __init__(self, name, options=None):
                              " FloatOptions.")
         NumericStatsMixin.__init__(self, options)
         BaseColumnPrimitiveTypeProfiler.__init__(self, name)
-        self.precision = 0
+
+        self.precision = {
+            'min': None,
+            'max': None,
+            'mean': None,
+            'var': None,
+            'std': None,
```

Comment:
are both these needed to calculate at run time? i would assume std is only needed at profile along with margin_of_error, hence I'm not sure they need storage so much as calc when needed. Could save some speed, but maybe not much.